src/api/app/controllers/person/token_controller.rb

```diff
@@ -5,12 +5,13 @@ class TokenController < ApplicationController
 
     before_action :set_user
     before_action :validate_operation, only: [:create]
+    after_action :verify_authorized
 
     # GET /person/<login>/token
     def index
-      authorize @user, :show?
+      authorize @user, :update?
```

Comment:
Why `update?` if its a read?

Reply:
Not sure what you mean. The name of the policy method?

Reply:
Yes

Reply:
Because you should only be able to see the tokens if you can change them.

Reply:
Makes sense